Components/GridPostsTeaser/_style.scss

```diff
@@ -0,0 +1,148 @@
+[is='flynt-grid-posts-teaser'] {
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    padding: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
+
+  .preContent {
+    @include reset-content-margins;
+
+    margin-bottom: 4.375rem;
+    text-align: center;
+  }
+
+  .postContent {
+    margin-top: $container-padding-mobile + 10;
+    text-align: center;
+
+    @media (min-width: $breakpoint-tablet) {
+      margin-top: $container-padding-tablet + 10;
+    }
+
+    @media (min-width: $breakpoint-desktop) {
+      margin-top: $container-padding-desktop + 10;
+    }
+  }
+
+  .grid {
+    @include reset-list;
+    @include grid(1, center);
+
+    @for $i from 2 through 4 {
+      &--length#{$i} {
+
+        @media (min-width: $breakpoint-tablet) {
+          @include grid(2, center);
+        }
+
+        @media (min-width: $breakpoint-desktop) {
+          @include grid($i, center);
+        }
+      }
+    }
+
+    &-item {
+      display: flex;
+      max-width: $content-max-width;
+    }
+  }
+
+  .content {
+    box-shadow: 0 1px 26px 0 rgba(0, 0, 0, 0.06);
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    text-decoration: none;
+    width: 100%; // IE11 bug fix
+
+    @media (min-width: $breakpoint-mobile-horizontal) {
+      transition: box-shadow 0.2s ease-in;
+    }
+
+    &:hover {
+      @media (min-width: $breakpoint-mobile-horizontal) {
+        box-shadow: none;
```

Comment:
Nice idea! Personally, I don't like it though. I feel it should be rather the other way around, that hovering over it, the shadow would get stronger and the item bigger. But right now, I would rather stick with no hover (as I don't have a better idea).